### DISCUSSION TITLE
Can not generate generate List<Inner Class> in class

### DISCUSSION THREAD

**Discussion by RameshwariH** [ozlerhakan/poiji#240] [Q&A]
Hi, Facing similiar issue like https://github.com/ozlerhakan/poiji/issues/190

public class ModifyCustomerRequest  {

    @ExcelCellName("accNum")
    public String accNum;

    public List<DocumentDetail> documentDetails;
}

public class DocumentDetail {

    @ExcelCellName("documentType")
    public int documentType;

    @ExcelCellName("documentStatus")
    public int documentStatus;
}

Can you please tell us how to acheive this?

**Comment by ozlerhakan** [comment]
Hi @RameshwariH ,

Poiji does not support this functionality, do you want to take it to make this possible?

**Reply by RameshwariH** [reply]
Hi @ozlerhakan, 
Is there any workaround to make it work?

**Reply by ozlerhakan** [reply]
currently we don't have @RameshwariH
